doc/5.reference/expr-help.pd

```diff
@@ -1,4 +1,4 @@
-#N canvas 275 90 942 546 12;
```

Comment:
the changes to *int-help* have removed important examples, namely the truncating/rounding towards zero.

i think this is a much more important property than being able to send to a `[receive]` or `[value]`, so you shouldn't remove this information.

Reply:
here's a new patch https://github.com/pure-data/pure-data/compare/master...porres:patch-1

Reply:
By the way, here's another simple and new update, related to this one https://github.com/pure-data/pure-data/pull/395